### ISSUE TITLE
Create curves

### ISSUE THREAD

**Issue by zkessin** [andrewtimberlake/elixir-pdf#23]
A way to create bezier curves would be very useful for some stuff I am working on (I would be willing to collaborate on a pull request)

**Comment by andrewtimberlake** [comment]
A pull request would be welcome

**Comment by zkessin** [comment]
I am working on it, the drawing seems to be in page.ex, is that correct?

**Comment by andrewtimberlake** [comment]
@zkessin That’s correct. Most operations are passed through Document to the current page and the actual writing/drawing is done on the page.
